fili-core/src/main/java/com/yahoo/bard/webservice/table/PhysicalTable.java

```diff
@@ -58,10 +60,12 @@ public PhysicalTable(
         this.workingIntervals = Collections.synchronizedMap(new HashMap<>());
         this.logicalToPhysicalColumnNames = Collections.unmodifiableMap(logicalToPhysicalColumnNames);
         this.physicalToLogicalColumnNames = Collections.unmodifiableMap(
-                this.logicalToPhysicalColumnNames
-                        .entrySet()
-                        .stream()
-                        .collect(StreamUtils.toLinkedMap(Map.Entry::getValue, Map.Entry::getKey))
+                this.logicalToPhysicalColumnNames.entrySet().stream().sequential().collect(
```

Comment:
I think removing the `sequential()` here is fine right? Will remove it in the next commit if no one complains.